featuretools/primitives/standard/aggregation_primitives.py

```diff
@@ -127,7 +187,17 @@ def get_function(self):
 
 
 class NUnique(AggregationPrimitive):
-    """Returns the number of unique categorical variables."""
+    """Finds the number of unique values.
+
+    Description:
+        Given a list of values, return the number of unique
+        values. Ignores `NaN` values.
+
+    Examples:
+        >>> num_unique = NUnique()
+        >>> num_unique([1, 2, 3, 4, 4, 5, None])
```

Comment:
let's make this example strings since input is discrete

Reply:
Fixed to strings (using colors)